nft-market/README.md

```diff
@@ -0,0 +1,38 @@
+This is a [Next.js](https://nextjs.org/) project bootstrapped with [`create-next-app`](https://github.com/vercel/next.js/tree/canary/packages/create-next-app).
```

Comment:
Update readme with project explanation and how does it work and how to start it

Reply:
Updated the README file